contracts/sources/reserve_config.move

```diff
@@ -57,6 +63,19 @@ module suilend::reserve_config {
         fields: Bag
     }
 
+    struct EmodeConfig has store {
```

Comment:
lets just use a VecMap? key is the pair reserve array index, value is the open/close ltv percent

Reply:
done